addons/policies/bootstrap_entity_policies.json

```diff
@@ -2185,7 +2186,8 @@
                 [
                     "admin",
                     "service-account-atlan-argo",
-                    "service-account-atlan-backend"
+                    "service-account-atlan-backend",
+                    "atlan-governance-workflows"
```

Comment:
This will allow atlan-governance-workflows user to CRUD ANY asset in Atlan, just making sure you are aware of it

Reply:
Correct, governance workflows will require CRUD Entity permissions for all change management usecases.